.github/CODEOWNERS

```diff
@@ -628,6 +628,7 @@ peps/pep-0746.rst  @JelleZijlstra
 peps/pep-0749.rst  @JelleZijlstra
 # ...
 peps/pep-0747.rst  @JelleZijlstra
+peps/pep-0750.rst  @jimbaker @gvanrossum @pauleveritt @lysnikolaou
```

Comment:
Only people with commit access can be added here (in this case, Guido and Lysandros).